storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -34,6 +34,12 @@ include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 include::modules/persistent-storage-csi-vsphere-stor-policy.adoc[leveloffset=+1]
 
+[NOTE]
+====
+If you want to update your cluster to {product-title} 4.10 or later and have previously installed an upstream vSphere CSI Operator Driver, you must remove the upstream driver before updating the cluster.
+====
+include::modules/persistent-storage-csi-vsphere-install-issues.adoc[leveloffset=+1]
```

Comment:
See above - we will be blocking upgrades to 4.11 not 4.10.

Reply:
Consider: 
```If you have planning to update or recently updated to 4.10 the Cluster Storage Operator may enter a `Degraded` state, due to previously installed an upstream vSphere CSI Operator Driver. To remedy this state you must remove the upstream driver before updating the cluster to 4.11.```

Reply:
The cluster will not enter `Degraded` state due to an upstream CSI driver. It will enter `upgradeable=false` state though.